libsodium-sys/build.rs

```diff
@@ -6,19 +6,19 @@ extern crate libc;
 #[cfg(target_env = "msvc")]
 extern crate vcpkg;
 
-#[cfg(not(target_env = "msvc"))]
 extern crate libflate;
-
-#[cfg(not(target_env = "msvc"))]
-extern crate tar;
-
 extern crate pkg_config;
+extern crate tar;
 
-use std::env;
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
 
 static VERSION: &'static str = "1.0.18";
 
 fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
Basically it makes it so that cargo is only rerun if build.rs is changed as opposed to any file in the root.
https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script

Reply:
But that being said, its probably not the behaviour we want so ill remove it.